README.md

```diff
@@ -22,8 +22,9 @@ Name | Possible Values | Description
 `issuingPublicKey` | Stellar public key | Public key of issuing account
 `assetCode` | `string` | Asset code of token to sell, ex. `TOKE`
 `price` | `string` | Maximum price of 1 `assetCode` token (in `BTC` or `ETH` depending which `start*` method is used)
-`bifrostURL` | `string` | URL of Bifrost server
-`horizonURL` | `string` | URL of Horizon server (_do not use SDF's servers!_)
+`bifrostURL`  | `string`  | URL of Bifrost server
+`horizonURL`  | `string`  | URL of Horizon server (_do not use SDF's servers!_)
+`preSaleMode` | `boolean` | (Optional) If set to `true`, tokens will not be sent to immediately to the user
```

Comment:
```diff
- tokens will not be sent to immediately to the user
+ tokens will not be sent immediately to the user
```
But I think it's not 100% clear. Maybe something like:
```
If set to `true`, BTC/ETH tokens will not be traded to `assetCode`. User will end up with BTC/ETH in their Stellar account.
```
And we should probably add a small note to `Bifrost.PurchasedEvent` event description like: "Will not be triggered in `preSaleMode`".

Reply:
Is `Bifrost.PurchasedEvent` in the `index` file? or on the `bifrost` server

Reply:
No, I mean in the docs (events table) here.

Reply:
okay updated